src/AboutDialog.cpp

```diff
@@ -1129,7 +1134,29 @@ wxT("OUT OF THE USE OR INABILITY TO USE THE PROGRAM (INCLUDING BUT NOT LIMITED\n
 wxT("TO LOSS OF DATA OR DATA BEING RENDERED INACCURATE OR LOSSES SUSTAINED BY\n")
 wxT("YOU OR THIRD PARTIES OR A FAILURE OF THE PROGRAM TO OPERATE WITH ANY OTHER\n")
 wxT("PROGRAMS), EVEN IF SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE\n")
-wxT("POSSIBILITY OF SUCH DAMAGES.\n"));
+wxT("POSSIBILITY OF SUCH DAMAGES.\n");
+
+#if defined(HAS_PRIVACY_POLICY)
+   /* i18n-hint: For "About Audacity..." Title for Pivacy Policy section */
+   const auto privacyPolicyTitle = XC("PRIVACY POLICY", "about dialog");
+
+   const auto privacyPolicyText = XO(
+      "App update checking and error reporting require network access. "
+      "These features are optional. See our [[https://www.audacityteam.org/about/desktop-privacy-notice/|Privacy Policy]] "
+      "for more information.");
```

Comment:
I would want to leave markup out of an internationalized string, just to make sure there is no error in any translation catalog.

Replace all that with a %s.  Then make another `XO(our Privacy Policy)` with i18n-hint that it is object of "see".

Why put "our" inside the i18n-hint?  Because some languages (like Finnish) want it to be a grammatical suffix!